src/styles/components/typeahead/styles.less

```diff
@@ -4,14 +4,32 @@
 
     .typeahead {
 
+      .sr-only,
+      .rbt-input-hint {
+        display: none;
+      }
+
+      .dropdown-menu li {
+        padding-left: 0;
+
+        &:before {
+          display: none;
+        }
+      }
+
+      .dropdown-item {
+        &:extend(.dropdown-menu-list li);
```

Comment:
I have found no good way to change the markup. The propose some, but it seems pretty cumbersome in the end. Could you explain the value this will bring to me?

Reply:
It would mean you don't need any of these styles except the `.sr-only, .rbt-input-hint` styles.
I'll leave it up to you to decide if the effort is worth removing a few lines of CSS :)